host-configs/corona-toss_4_x86_64_ib-clang@14.0.0_hip.cmake

```diff
@@ -60,13 +62,13 @@ set(ENABLE_MPI ON CACHE BOOL "")
 
 set(ENABLE_HIP ON CACHE BOOL "")
 
-set(HIP_ROOT_DIR "/opt/rocm-4.5.2/hip" CACHE STRING "")
+set(HIP_ROOT_DIR "/opt/rocm-5.1.0/hip" CACHE STRING "")
 
-set(HIP_CLANG_PATH "/opt/rocm-4.5.2/hip/../llvm/bin" CACHE STRING "")
+set(HIP_CLANG_PATH "/opt/rocm-5.1.0/hip/../llvm/bin" CACHE STRING "")
 
 set(CMAKE_HIP_ARCHITECTURES "gfx906" CACHE STRING "")
 
-set(CMAKE_EXE_LINKER_FLAGS "-Wl,--disable-new-dtags -L/opt/rocm-4.5.2/hip/../llvm/lib -L/opt/rocm-4.5.2/hip/lib -Wl,-rpath,/opt/rocm-4.5.2/hip/../llvm/lib:/opt/rocm-4.5.2/hip/lib -lpgmath -lflang -lflangrti -lompstub -lamdhip64 -L/opt/rocm-4.5.2/hip/../lib64 -Wl,-rpath,/opt/rocm-4.5.2/hip/../lib64 -lhsakmt -lamd_comgr" CACHE STRING "")
+set(CMAKE_EXE_LINKER_FLAGS "-Wl,--disable-new-dtags -L/opt/rocm-5.1.0/hip/../llvm/lib -L/opt/rocm-5.1.0/hip/lib -Wl,-rpath,/opt/rocm-5.1.0/hip/../llvm/lib:/opt/rocm-5.1.0/hip/lib -lpgmath -lflang -lflangrti -lompstub -lamdhip64 -L/opt/rocm-5.1.0/hip/../lib64 -Wl,-rpath,/opt/rocm-5.1.0/hip/../lib64 -lhsakmt -lamd_comgr" CACHE STRING "")
```

Comment:
Do you know if these can be removed now? (I know they were required for 4.5.2)

Reply:
> Do you know if these can be removed now? (I know they were required for 4.5.2)

These flags are still necessary ( I gave it a try without the flags, ran into errors). The quickstart guide for nevada still recommends these flags when using non-tce wrapped compilers.